src/components/QuantitySelector.js

```diff
@@ -0,0 +1,58 @@
+import styled from "styled-components";
+import { FaPlus, FaMinus } from "react-icons/fa";
+
+const QuantityButton = styled.button`
+    background: ${(p) => p.theme.colors.background};
+    color: ${(p) => p.theme.colors.text};
+    border: none;
+`
+
+const handleDecrease = (value, setValue) => {
+    if (value === 0) return; 
+    setValue(value - 1)
```

Comment:
what if value is < 0?

I think it would be cleaner to have something like 
```suggestion
    if (value > 0) setValue(value - 1); 
```

Reply:
value can't ever be less than 0 :) but yes this is good! i'll change it